src/features/projectsV2/ProjectsMap/index.tsx

```diff
@@ -92,22 +92,28 @@ function ProjectsMap(props: ProjectsMapProps) {
   const onClick = useCallback(
     (e) => {
       if (props.page !== 'project-details') return;
+      const hasNoSites =
+        singleProject?.sites && singleProject?.sites?.length === 0;
       const result = getPlantLocationInfo(plantLocations, mapRef, e.point);
```

Comment:
You could probably shorten this condition to 

`singleProject?.sites?.length === 0`